build.sbt

```diff
@@ -56,7 +56,7 @@ TwirlKeys.templateImports ++= Seq(
     "play.api.Play.current"
 )
 
-val awsVersion = "1.11.18"
+val awsVersion = "1.11.8"
```

Comment:
It was initially because of the deploy issue I had, but found the problem in the config file. I'll take this out to see if it's still working.